draft-decraeneginsberg-lsr-isis-fast-flooding.xml

```diff
@@ -346,113 +356,138 @@
 			use its local default value. That value SHOULD
 			be configurable on a per node basis and MAY be
 			configurable on a per interface basis.
-			</t> 
-			<section anchor="LSPReceptionWindow" title="LSP Burst Window sub-TLV">
-				<t>The LSP Burst Window sub-TLV advertises the maximum number of LSPs that the node can receive with no separation interval between LSPs.</t>
-				<t>Type: 1</t>
-				<t>Length: 4 octets</t>
-				<t>Value: number of LSPs that can be sent back to back</t>
+		</t>
+		<section anchor="LSPReceptionWindow" title="LSP Burst Window sub-TLV">
+			<t>The LSP Burst Window sub-TLV advertises the maximum number of LSPs that the node can receive with no separation interval between LSPs.</t>
+			<t>Type: 1</t>
+			<t>Length: 4 octets</t>
+			<t>Value: number of LSPs that can be sent back to back</t>
+		</section>
+		<section anchor="InterfaceLSPTransmissionInterval" title="LSP Transmission Interval sub-TLV">
+			<t>The LSP Transmission Interval sub-TLV advertises the minimum interval, in micro-seconds, between LSPs arrivals which can be received on this interface, after the maximum number of un-acknowledged LSPs has been sent.</t>
+			<t>Type: 2</t>
+			<t>Length: 4 octets</t>
+			<t>Value: minimum interval, in micro-seconds, between two consecutive LSPs sent after the burst window has been used</t>
+			<t>The LSP Transmission Interval is an advertisement of the receiver's steady-state LSP reception rate.</t>
+		</section>
+		<section anchor="LPP" title="LSPs Per PSNP sub-TLV">
+			<t>The sub-TLV LSP Per PSNP (LPP) advertises the number of received LSPs that triggers the immediate sending of a PSNP to acknowledge them.</t>
+			<t>Type: 3</t>
+			<t>Length: 2 octets</t>
+			<t>Value: number of LSPs acknowledged per PSNP</t>
+			<t>A node advertising this sub-TLV with a value LPP MUST send a PSNP once LPP LSPs have been received and need to be acknowledged./t>
 			</section>
-			<section anchor="InterfaceLSPTransmissionInterval" title="LSP Transmission Interval sub-TLV">
-				<t>The LSP Transmission Interval sub-TLV advertises the minimum interval, in micro-seconds, between LSPs arrivals which can be received on this interface, after the maximum number of un-acknowledged LSPs has been sent.</t>
-				<t>Type: 2</t>
-				<t>Length: 4 octets</t>
-				<t>Value: minimum interval, in micro-seconds, between two consecutive LSPs sent after the burst window has been used</t>
-				<t>The LSP Transmission Interval is an advertisement of the receiver's steady-state LSP reception rate.</t>
+			<section anchor="Flags" title="Flags sub-TLV">
+				<t>The sub-TLV Flags advertises a set of flags. Flags are standardized and IANA registered.</t>
+				<t>Type: 4</t>
+				<t>Length: Indicates the length in octets (1-8) of the Value field. The length SHOULD be the minimum required to send all bits that are set.</t>
+				<t>Value: List of flags.</t>
+
+				<artwork align="left" pn="Flags">
```

Comment:
This needs to be wrapped in t and figure tags.  Xml2rfc does not understand 'pn'.